test/config.js

```diff
@@ -0,0 +1,11 @@
+const config = {
+  database: {
+    datatype: "mysql",
+    host: "localhost",
+    name: "test",
+    user: "root",
+  },
+  endpoint: "/auth",
+};
+
+export default () => config;
```

Comment:
export the config object directly as default or name your export and ignore the eslint rule, but do not return an anonymous function as a default export.